### ISSUE TITLE
build: assets are not copied correctly

### ISSUE THREAD

**Issue by dsebastien** [NationalBankBelgium/stark#415]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#got-a-question-or-problem
</code></pre>

## Current behavior
The production build (at least) does not copy all the necessary assets.

## Expected behavior
The configurations have to be reviewed and fixed for all stark packages, the starter and the showcase.
Once done, running a dev or production build should end up with consistent results in the dist folder.
We should also take the occasion to clean things up (duplicate assets, things that should be in stark-ui, ...).

**Comment by SuperITMan** [comment]
Actually, I think everything is working fine. But our config into angular.json is wrong...
Our current content is the following:
* starter
  * assets
    * folder1
    * folder2
  * src
    * assets
      * folder3
      * folder4

And in the generated dist:
* dist
  * assets
    * folder3
    * folder4

In our angular.json, we have
```json
"build": {
  // ...
  "options": {
    // ...
    "assets": [
      {
        "glob": "**/*",
        "input": "assets",
        "output": "./assets"
      }
    ]
}
```

But if we want to copy also **folder1** and **folder2**, we have to add the following assets (for info: rootDir is "src")
```json
{
  "glob": "**/*",
  "input": "../assets",
  "output": "./assets"
},
```

**Comment by RobbyDeLaet** [comment]
I discussed the location of the **assets** folder with @christophercr yesterday while I was cleaning up the **Starter** application.

For a developer it might be confusing that there are two **assets** folders in the same project. At least it adds unnecessary complexity. Besides that, it adds also the possibility of unwanted behavior when a file with the same name but with different content is stored in both folders.

Therefore I have only kept the **assets** folder underneath the **src** folder, because that is the default location. The contents of the other **assest** folder in moved into this one. In **stark-ui** the **angular.json** file is modified so that the content of the **assets** folder is copied into the **src/assets** folder of the client app ("output": ".**src/**assets/stark-ui/").

**Comment by SuperITMan** [comment]
> Therefore I have only kept the assets folder underneath the src folder, because that is the default location. The contents of the other assest folder in moved into this one. In the stark-ui package, the angular.json file is modified so that the content of the assets folder is copied into the src/assets folder of the client app ("output": ".src/assets/stark-ui/").

Normally, the output should be **./assets**, not **./src/assets/stark-ui** which is a no-sense...
The files from stark-ui or stark-core are copied during the build process, not during the install of the packages.

**Comment by dsebastien** [comment]
@RobbyDeLaet please also adapt the Showcase while you're at it.

I agree that we should avoid having more than 1 "assets" folder per package/app, otherwise it becomes messy and hard to follow.

The distinction between assets & assets-base is still meaningful I think, both for our packages & the apps
